.github/workflows/ci.yml

```diff
@@ -8,16 +8,16 @@ jobs:
     build:
         runs-on: ubuntu-22.04
         name: "python ${{ matrix.python-version }} on ${{ matrix.backend }}"
-        # https://github.com/pypy/pypy/issues/4956
-        continue-on-error: "${{ matrix.python-version == 'pypy-3.10' }}"
         strategy:
             matrix:
                 # If you change one of these, be sure to update:
                 # - /tox.ini:[gh-actions]
                 # - /setup.cfg:[mypy]
                 # If adding new python versions, consider also updating
                 # python version in .readthedocs.yaml
-                python-version: [pypy-3.10, '3.10', '3.11', '3.12']
+                # TODO: Add back pypy here and in release.yml once:
+                # https://github.com/pypy/pypy/issues/4956 is resolved
+                python-version: ['3.10', '3.11', '3.12']
```

Comment:
Are we all agreed that we're now dropping pypy 3.10 (as opposed to just allowing it to fail)?

No objections from me!

Reply:
My thought process was that for releases it would be nice that CI doesn't fail. After the release we can re-enable it

Reply:
Fine with me, though can you rebase and drop the continue-on-error I just added as well?

Reply:
If we do get some movement on that issue, we can always drop the note from the CHANGELOG and re-add pypy to CI.

Reply:
Oh, derp. You had it!